docs/BR.md

```diff
@@ -2,6 +2,7 @@
 title: Baseline Requirements for the Issuance and Management of Publicly-Trusted TLS Server Certificates
 subtitle: Version 2.0.5
 author:
```

Comment:
The front matter is in YAML, not Markdown.

It should be linted as YAML where this nestling would not be expected.

Reply:
I've removed the changes to the YAML frontmatter.